tests/phpunit/CRM/Member/Form/MembershipRenewalTest.php

```diff
@@ -379,12 +379,12 @@ public function testSubmitRecurCompleteInstant(): void {
     $log = $this->callAPISuccessGetSingle('MembershipLog', ['membership_id' => $membership['id'], 'options' => ['limit' => 1, 'sort' => 'id DESC']]);
     $this->assertEquals(CRM_Utils_Time::date($nextYear . '-01-01'), $log['start_date']);
     $this->assertEquals(CRM_Utils_Time::date($nextYear . '-01-31'), $log['end_date']);
-    $this->assertEquals(CRM_Utils_Time::date('Y-m-d'), $log['modified_date']);
+    $this->assertEquals(date('Y-m-d'), CRM_Utils_Time::date('Y-m-d', strtotime($log['modified_date'])));
```

Comment:
Is there a reason not to just change equals part to compare against the time too? We're using frozen time (the use of CRM_Utils_Time) in these tests so second rollovers won't be an intermittent issue. By removing CRM_Utils_Time and just using date() it's thawing out the time.

```suggestion
    $this->assertEquals(CRM_Utils_Time::date('Y-m-d H:i:s'), $log['modified_date']);
```

I haven't run that but given that the PR is about ADDING a time component, it seems like we would want to test that time is getting used.

Reply:
Fair enough. Why do you say it's using frozen time? I can see the class but there are various options and I don't see where the input is set to frozen?

Reply:
Ah you're right it just sets a fixed "starting" time that the universe is relative to. So if we do start getting second rollovers there is another method available to compare two numbers with a fudge factor: assertApproxEquals()

Reply:
Ok, I've updated to use assertApproxEquals with a tolerance of 20 seconds